driver/registry_sql.go

```diff
@@ -86,6 +86,7 @@ func (m *RegistrySQL) Init(ctx context.Context) error {
 			Pool:                      pool,
 			UseInstrumentedDriver:     m.Tracer(ctx).IsLoaded(),
 			InstrumentedDriverOptions: opts,
+			Unsafe:                    m.C.DbIgnoreUnknownTableColumns(),
```

Comment:
Wow, I didn't know the solution is that easy! I thought it needed much more work. Great stuff! :)